bot.py

```diff
@@ -351,35 +351,48 @@ def set_and_check_team(self, player, team):
 				status = params[1]
 				if self.own_pokemon(pokemon_data):
 					self.add_status(self.statuses, pokemon_name, status)
+					self.gs.set_status(GameState.Player.one, pokemon_name, status)
+
 				else:
 					self.add_status(self.opp_statuses, pokemon_name, status)
+					self.gs.set_status(GameState.Player.two, pokemon_name, status)
 
 			elif inp_type == '-start':
 				pokemon_data = params[0]
 				pokemon_name = self.get_pokemon(pokemon_data)
 				status = params[1]
 				if self.own_pokemon(pokemon_data):
 					self.add_status(self.statuses, pokemon_name, status)
+					self.gs.set_status(GameState.Player.one, pokemon_name, status)
+
 				else:
 					self.add_status(self.opp_statuses, pokemon_name, status)
+					self.gs.set_status(GameState.Player.two, pokemon_name, status)
 
 			elif inp_type == '-end':
 				pokemon_data = params[0]
 				pokemon_name = self.get_pokemon(pokemon_data)
 				status = params[1]
 				if self.own_pokemon(pokemon_data):
 					self.remove_status(self.statuses, pokemon_name, status)
```

Comment:
similar thing here, self.gs.remove_status could probably just be a part of self.remove_status

Reply:
Makes sense